modules/navbar.js

```diff
@@ -0,0 +1,44 @@
+const Navbar = () => {
+  const bookForm = document.getElementById('book-entry');
+  const bookStore = document.getElementById('bookStore');
+  const listClick = document.querySelector('#list');
+  const addClick = document.querySelector('#add-book');
+  const contactClick = document.querySelector('#contact-details');
+  const storeTitle = document.querySelector('#store-title');
+  const contacts = document.querySelector('#contact-box');
+
+  const showForm = () => {
+    if (bookStore.style.display === 'block') {
+      bookStore.style.display = 'none';
+      storeTitle.style.display = 'none';
+      contacts.style.display = 'none';
+    } else if (contacts.style.display) {
+      contacts.style.display = 'none';
+    }
+    bookForm.style.display = 'flex';
+  };
+
+  const showShelf = () => {
+    if (bookForm.style.display === 'flex') {
+      bookForm.style.display = 'none';
+    }
+    contacts.style.display = 'none';
+    bookStore.style.display = 'block';
+    storeTitle.style.display = 'block';
+  };
+
+  const contactUs = () => {
+    if (bookStore.style.display === 'block') {
+      bookStore.style.display = 'none';
+      storeTitle.style.display = 'none';
+    }
+    bookForm.style.display = 'none';
+    contacts.style.display = 'block';
+  };
+
+  contactClick.addEventListener('click', contactUs);
+  addClick.addEventListener('click', showForm);
+  listClick.addEventListener('click', showShelf);
+};
+
+export default Navbar;
```

Comment:
- [ ] Great work changing your functions to use the arrow syntax. However, I noticed that when I click on a link, the relevant part of the page does not show. Kindly fix this. 👍 

![chrome-capture-2023-7-14](https://github.com/Chimoneg27/Awesome-Books-ES6/assets/77106962/9cb23b70-200a-4a4b-b9b9-3675ac4738bb)